File: src/screens/HomeScreen.js
import {Button, Text, View} from 'react-native';

import React from 'react';

const HomeScreen = ({navigation}) => {
  return (
    <View>
      <Text>Welcome to FinWise</Text>
      <Button
        title="Get Started"
        onPress={() => navigation.navigate('AnotherScreen')} // Placeholder for navigation
      />
    </View>
  );
};

export default HomeScreen;

File: lib/screens/home_screen.dart
import 'package:flutter/material.dart';

/// A stateless widget representing the home screen.
class HomeScreen extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text('FinWise')),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Text('Welcome to FinWise - Your Financial Planning App'),
            ElevatedButton(
              onPressed: () {
                // Navigate to the login screen when the button is pressed.
                Navigator.pushNamed(context, '/login');
              },
              child: Text('Login'),
            ),
            ElevatedButton(
              onPressed: () {
                // Navigate to the registration screen when the button is pressed
                Navigator.pushNamed(context, '/register');
              },
              child: Text('Register'),
            ),
          ],
        ),
      ),
    );
  }
}
